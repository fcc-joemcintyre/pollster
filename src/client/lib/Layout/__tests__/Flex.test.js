/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import { Flex } from '..';

test ('Flex renders default component', () => {
  const component = renderer.create (
    <Flex />
  );
  const tree = component.toJSON ();
  expect (tree).toMatchSnapshot ();
});

test ('Flex renders wrap component', () => {
  const component = renderer.create (
    <Flex wrap />
  );
  const tree = component.toJSON ();
  expect (tree).toMatchSnapshot ();
});
