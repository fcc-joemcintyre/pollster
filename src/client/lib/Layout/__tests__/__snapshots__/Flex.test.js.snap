// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flex renders default component 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<div
  className="c0"
  wrap={null}
/>
`;

exports[`Flex renders wrap component 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

<div
  className="c0"
  wrap={true}
/>
`;
